xep-0292.xml

```diff
@@ -173,155 +175,160 @@
-    <p>&xep0060; provides a way to subscribe to events, and &xep0163; defines a pubsub profile for events associated with instant messaging (IM) accounts. If PEP is supported by an IM server, it can be used to automatically generate event notifications when a user's vCard is modified.</p>
-    <section3 topic='Location' anchor='self-pubsub-location'>
-      <p>The canonical location for notifications regarding a user's vCard is a pubsub node whose name is "urn:xmpp:vcard4".</p>
-    </section3>
+    <p>&xep0163; also provides a way to subscribe to events. It can be used to automatically generate event notifications when a user's vCard is modified.</p>
     <section3 topic='Subscribing to vCard Notifications' anchor='self-pubsub-subscribe'>
       <p>Let us imagine that Juliet wishes to receive the updates that Romeo publishes to his vCard. She has two options:</p>
       <ol>
-        <li>Implicitly subscribe by advertising support for "urn:xmpp:vcard4+notify" in her &xep0115; data. Romeo's PEP service then automatically sends vCard updates to her when it receives presence from her, until and unless she sends presence of type unavailable or stops advertising an interest in vCard updates. This is in accordance with XEP-0060, section 6.1.</li>
+        <li>Implicitly subscribe by advertising support for "urn:xmpp:vcard4+notify" in her &xep0115; data. Romeo's PEP service then automatically sends vCard updates to her when it receives presence from her, until and unless she sends presence of type unavailable or stops advertising an interest in vCard updates. This is in accordance with &xep0060;, section 6.1.</li>
         <li>Explicitly subscribe by sending a formal subscription request to the "urn:xmpp:vcard4" node at Romeo's JabberID. Romeo's PEP service might send her all vCard updates even if she is offline at the time (depending on service policies regarding presence integration).</li>
       </ol>
     </section3>
     <section3 topic='Receiving a vCard Notification'>
-      <p>Because Juliet has sent presence to Romeo including Entity Capabilities data that includes the "urn:xmpp:vcard4+notify" feature, Romeo's XMPP server will send a PEP notification to Juliet. The notification can include an XMPP message body for backward-compatibility with XMPP clients that are not pubsub-capable. This is in accordance with XEP-0060, second 6.1.7.</p>
+      <p>Because Juliet has sent presence to Romeo including Entity Capabilities data that includes the "urn:xmpp:vcard4+notify" feature, Romeo's XMPP server will send a PEP notification to Juliet.</p>
       <example caption="Receiving a vCard publication/update"><![CDATA[
   <message from='romeo@montague.lit'
            to='juliet@capulet.lit'
            type='headline'>
     <event xmlns='http://jabber.org/protocol/pubsub#event'>
       <items node='urn:xmpp:vcard4'>
-        <item id='current'/>
+        <item id='1e3fdd90-f542-41d3-82ae-33d183467a7a'/>
```

Comment:
Why change this? [Asked in xsf@](https://logs.xmpp.org/xsf/2022-11-09?p=h#2022-11-09-4dffc3f70d5f479d)

Reply:
I thought it was more clear for the ID here to match the ID in the publication example above, as this is an example notification about that publication, in effect.

Reply:
But shouldn't the publication also use `id='current'` to avoid to accumulate ever more items?

Reply:
Probably `pubsub#max_items` should be set to `1` for the node.

Reply:
(tho you'd probably limit the node to 1 item)

Reply:
Did this conversation get resolved?

Reply:
Dunno, should it follow the [singleton](https://xmpp.org/extensions/xep-0060.html#impl-singleton) pattern or nah?

Reply:
Given what the XEP already says

> Note: There is no payload, because this is a pure notification (the receiver needs to retrieve the vCard using an IQ-get as described earlier).

I think as it was written (using `current`) is basically not usable because you will always get a notification on connect that says "vcard is current" which... is just obviously true and tells you no information?